VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -978,11 +978,14 @@ subroutine vom_init_vegpar ()
 
 !     * Set vegetation parameters
 
-      q_md   = o_pct * i_mdtf + o_mdstore
+      q_md   = o_cai * i_mdtf + o_mdstore
       q_mqx  = q_md * i_mqxtf
       mqtnew = 0.95d0 * q_mqx                  ! initial wood water storage
       mqtold = mqtnew
       rsurftnew(:) = 0.d0
+      lai_lt(:) = 2.5d0 * (/1.0d0-i_incrlait,1.0d0,1.0d0+i_incrlait/)
+      lai_lg(:) = 2.5d0 * (/1.0d0-i_incrlaig,1.0d0,1.0d0+i_incrlaig/)
+
```

Comment:
Why is the 2.5 still here? What does it mean?

Reply:
This is the subroutine that initializes everything, I setted the initial LAI-value to 2.5. Actually we should move this probably to the namelist as well, but for now I hard-coded it to see if it worked anyway.